Dockerfile

```diff
@@ -3,11 +3,17 @@ FROM wordpress:latest
 ARG woocommerce_version
 
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends unzip wget gettext \
-    && wget https://downloads.wordpress.org/plugin/woocommerce.${woocommerce_version}.zip -O /tmp/temp.zip \
+    && apt-get install -y --no-install-recommends unzip wget gettext vim \
+    && wget https://downloads.wordpress.org/plugin/woocommerce.${woocommerce_version}.zip -O /tmp/woocommerce.zip \
+    && wget https://downloads.wordpress.org/plugin/relative-url.0.1.7.zip -O /tmp/relative-url.zip \
     && cd /usr/src/wordpress/wp-content/plugins \
-    && unzip /tmp/temp.zip \
-    && rm /tmp/temp.zip \
+    && unzip /tmp/woocommerce.zip \
+    && unzip /tmp/relative-url.zip \
+    && rm /tmp/woocommerce.zip \
+    && rm /tmp/relative-url.zip \
     && rm -rf /var/lib/apt/lists/*
 
+RUN mkdir -p /var/www/html/wp-content/uploads/wc-logs/ \
```

Comment:
For the plugin to write to log files, the folder needs to be created and the WP user needs to have access (www-data is the WP user)